# Header of the published topic
Header header


# The header of the input image
Header imageHeader
# a representor vector
float32[]   representor